### DISCUSSION TITLE
How to run LMDeploy gradio but for vision models

### DISCUSSION THREAD

**Discussion by bks5881** [InternLM/lmdeploy#1718] [Q&A] [answered]
I can run lmdeploy serve MODEL_NAME  gradio, but this only launches text only gradio server. How to launch gradio server with UI for multimodel, for example llava?

**Comment by lvhan028** [comment]
@AllentDan could you do the favor?

**Comment by AllentDan** [comment]
Just launch the vl model with the exact same command as the llm models. For example: `lmdeploy serve gradio deepseek/deepseek-vl-1.3b-chat`

**Reply by bks5881** [reply]
will this also support adding images to the prompt?

**Reply by AllentDan** [reply, marked as answer]
there is a button there for you to upload images.

**Reply by BigWhiteFox** [reply]
> will this also support adding images to the prompt?
U can use the api_serve command to deploy llm. Create an ui what u want and then convey message in Openai format between user and llm. This may be more flexible with the help of gradio package.
